babel.config.js

```diff
@@ -25,7 +25,6 @@ const defaultAlias = {
   '@mui-internal/test-utils': resolveAliasPath(
     './node_modules/@mui/monorepo/packages/test-utils/src',
   ),
-  'typescript-to-proptypes': '@mui/monorepo/packages/typescript-to-proptypes/src',
```

Comment:
What is the reasoning behind this removal
Do we intend to keep the `@mui/monorepo/packages/typescript-to-proptype` import in our codebase intead?

Reply:
There are no imports from `typescript-to-proptypes` directly, so it's not needed.
I'm going to use the proptypes generator from Core, so there won't be a need anymore to import from @mui/monorepo/packages/typescript-to-proptype either.

Reply:
What would be the import we would use in the end ?
Should we rename the `typescript-to-proptypes` folder to something more coherent with our other packages? `@mui-internal/proptypes-generator` for instance).

Reply:
I changed the imports to be `typescript-to-proptypes` in #10418 but wasn't able to merge because of some other issues.
IMHO having `@mui/monorepo` imports is to be avoided because it means that anything in the core repo might be used by other teams.

Reply:
Ultimately you won't import from the monorepo but from the npm package. Then, the import will be @mui-internal/typescript-to-proptypes.
I suppose I can use the new import today and leave the Babel alias so when we change the monorepo import to a package, we'll only need to change the Babel config.